packages/recommend-core/src/types/InitialResults.ts

```diff
@@ -0,0 +1,5 @@
+import { ProductRecord } from './ProductRecord';
+
+export type InitialResults<TObject> = {
```

Comment:
Related to the naming again, can we name this type the same as the prop name? (`InitialState`)

Reply:
✅